app/styles/toggle-buttons.less

```diff
@@ -0,0 +1,26 @@
+.oss-toggle-buttons-container {
+  .border-radius-default;
+  .background-color-gray-100;
+  .text-color-default;
+
+  display: flex;
+  flex: 1;
+  height: 30px;
+  padding: 3px;
+
+  .oss-toggle-buttons-btn {
+    .border-radius-default;
+
+    display: flex;
+    flex: 1;
+    justify-content: center;
+    align-items: center;
+    padding:3px;
+
+    &--selected {
+      .background-color-white;
+
+      box-shadow: 0px 2px 3px 0px var(--color-gray-300);
```

Comment:
on the Figma design, there is usually a comment above the box-shadow rule that says which one it is (XS, MD, etc.). You can then define it here using the right CSS variable. Here it seems to be `var(--box-shadow-xs)`